internal/cloud/realm/dependencies.go

```diff
@@ -18,10 +19,43 @@ const (
 	dependenciesPathPattern        = appPathPattern + "/dependencies"
 	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
 	dependenciesDiffPathPattern    = dependenciesPathPattern + "/diff"
+	dependenciesStatusPathPattern  = dependenciesPathPattern + "/status"
 
 	paramFile = "file"
 )
 
+// DependencyInstallation is used to get information from a dependencies status request
+type DependencyInstallation struct {
+	Status        DependenciesStatus `json:"status"`
+	StatusMessage DependenciesStatus `json:"status_message"`
```

Comment:
[question] why do we have this field, it does not seem to be used anywhere?  if we do need it, I'm guessing it wants to be of type `string` rather than `DependenciesStatus`

Reply:
that's true it's meant to be a string and will be used in the spinner based on earlier feedback from Andy, was still working on it so this was more of in an intermediary stage.